gamemodes/terrortown/gamemode/shared/hud_elements/base_elements/hud_element_base/cl_init.lua

```diff
@@ -730,6 +730,6 @@ function HUDELEMENT:LoadData()
 	end
 
 	for k, v in pairs(loadedData) do
-		self[k] = v or self[k]
```

Comment:
I don't think this is correct, since now any defined key will just be kept and never be overridden by the loaded data